### PR TITLE
Move the radanalytics-pyspark base image to python s2i

### PR DESCRIPTION
**Pull request by tmckayus**

### CODE REVIEW

**Comment by tmckayus** [comment]
Closed #25 and re-opened as #26 with alterations

File: pyspark/Dockerfile [hunk 1]
```diff
@@ -15,13 +11,23 @@ LABEL io.k8s.description="Platform for building a radanalytics pyspark app" \
...
+RUN yum install -y tar java && \
+    yum clean all
+
+RUN cd /opt && \
+    curl https://dist.apache.org/repos/dist/release/spark/spark-2.0.0/spark-2.0.0-bin-hadoop2.7.tgz | \
+        tar -zx && \
+    ln -s spark-2.0.0-bin-hadoop2.7 spark
+
+ENV PATH=$PATH:/opt/spark/bin
+ENV SPARK_HOME=/opt/spark
```
Comment (mattf) [inline pyspark/Dockerfile]: how does the app find pyspark imports?

do we need ENV PYTHONPATH=/opt/spark/pyspark here? do we need py4j install?

or are those all handled by spark-submit?

Comment (tmckayus) [inline pyspark/Dockerfile]: In my simple example apps, "from pyspark import ..." works so I suspect that spark-submit is adding it to the python path. I will attempt to confirm.

Comment (mattf) [inline pyspark/Dockerfile]: i expect it is too. lgtm

Comment (tmckayus) [inline pyspark/Dockerfile]: fyi, standard python path from a radanalytics-pyspark app pod:

$ python -c "import sys; print '\n'.join(sys.path)"                       
                                                                                
/opt/rh/python27/root/usr/lib64/python27.zip                                    
/opt/rh/python27/root/usr/lib64/python2.7                                       
/opt/rh/python27/root/usr/lib64/python2.7/plat-linux2                           
/opt/rh/python27/root/usr/lib64/python2.7/lib-tk                                
/opt/rh/python27/root/usr/lib64/python2.7/lib-old                               
/opt/rh/python27/root/usr/lib64/python2.7/lib-dynload                           
/opt/rh/python27/root/usr/lib64/python2.7/site-packages                         
/opt/rh/python27/root/usr/lib/python2.7/site-packages  

Path from inside a pyspark app run from spark-submit on the same pod:

/opt/app-root/src
/tmp/spark-d992dcf5-9b54-4c6f-b57d-581742d817c6/userFiles-9d2c9d01-7ce2-4d72-9ef7-5146ed811f67
/opt/spark/python/lib/pyspark.zip
/opt/spark/python/lib/py4j-0.10.1-src.zip
/opt/spark-2.0.0-bin-hadoop2.7/jars/spark-core_2.11-2.0.0.jar
/opt/rh/python27/root/usr/lib64/python27.zip
/opt/rh/python27/root/usr/lib64/python2.7
/opt/rh/python27/root/usr/lib64/python2.7/plat-linux2
/opt/rh/python27/root/usr/lib64/python2.7/lib-tk
/opt/rh/python27/root/usr/lib64/python2.7/lib-old
/opt/rh/python27/root/usr/lib64/python2.7/lib-dynload
/opt/rh/python27/root/usr/lib64/python2.7/site-packages
/opt/rh/python27/root/usr/lib/python2.7/site-packages

Comment (mattf) [inline pyspark/Dockerfile]: thanks

**Pull request merged**